src/components/tire/AddTireCheckPage.module.css

```diff
@@ -0,0 +1,182 @@
+.button,
+.spareTireButton {
+  width: 100%;
+}
+
+.spareTireButton {
+  background-color: rgb(37, 147, 238);
+  color: white;
+}
+
+.tireGrid {
+  display: grid;
```

Comment:
How it works is that the top global tire pressure is just a flex box, but the previous tire pressures are laid out via a grid layout.